scripts/merge-forward.sh

```diff
@@ -8,7 +8,11 @@ git checkout -B v2-main origin/v2-main
 
 # Some package rules differ between v1 and v2, most notably which packages can be public vs private.
 # These differences are fixable via 'pkglint', so we run that and commit the delta (if any).
+yarn install --frozen-lockfile
+# Build pkglint and its dependencies (e.g., eslint-plugin-cdk), then run.
+cd tools/pkglint && ../../scripts/buildup && cd ../../
```

Comment:
Can we instead update the `pkglint` script [here](https://github.com/aws/aws-cdk/blob/b16a8d3079d1d293ce127d616f82b54cb869f2e5/package.json#L9) to `lerna --scope pkglint --include-dependencies run build`?

Reply:
Already tried that (three iterations ago):

```
% lerna --scope pkglint --include-dependencies run build
ERR! lerna Unknown arguments: scope, include-dependencies, includeDependencies, build
```

...actually...

```
% lerna run build --scope pkglint --include-dependencies
info cli using local version of lerna
lerna notice cli v3.22.1
lerna notice filter including "pkglint"
lerna notice filter including dependencies
lerna info filter [ 'pkglint' ]
lerna info Executing command in 2 packages: "yarn run build"
...
```

That works! Odd that the first version doesn't, but the second does.

Reply:
Good you found that.